src/Engine/ProtoCore/Reflection/GraphicDataProvider.cs

```diff
@@ -141,50 +141,31 @@ internal List<IGraphicItem> GetGraphicItems(DSASM.StackValue svData, RuntimeCore
             return null;
         }
 
-        //Store marshaller for repeated calls to GetCLRObject.  Used for short-circuit of re-allocation of marshaller / interpreter object.
-        private static Interpreter interpreter;
-        private static ProtoFFI.FFIObjectMarshaler marshaler;
-        
         internal object GetCLRObject(StackValue svData, RuntimeCore runtimeCore)
         {
             if (null == runtimeCore.DSExecutable.classTable)
                 return null;
 
-            //The GetCLRObject function is typically utilized to retrieve a ClrObject from a StackValue of type pointer.
-            //There is an edge cases for pointers where the pointer references a non CLR object.  This code
-            //checks for this edge case by verifying that the requested StackValue pointer is associated with an
-            //imported library.  An example is the "Function" pointer which does not have an associated CLRObject.
-            //In that case, the return value should be null.
-            var classNode = runtimeCore.DSExecutable.classTable.GetClassNodeAtIndex(svData.metaData.type);
```

Comment:
I believe `GetClassNodeAtIndex` was also introduced as part of the previous fixes. I suppose it's no longer needed as this line of code has been removed. Can the function, therefore, be removed?

Reply:
Yep it can be... I will file a follow up to re-look at this optimization.  And we can either to fix the optimization or clear up that function or